www/js/lib/uiUtil.js

```diff
@@ -439,13 +449,6 @@ define(rqDef, function() {
         footer.classList.add(contentTheme || '_light');
         // Embed a reference to applied theme, so we can remove it generically in the future
         htmlEl.dataset.theme = theme;
-        // Hide any previously displayed help
-        var oldHelp = document.getElementById(oldTheme + '-help');
-        if (oldHelp) oldHelp.style.display = 'none';
-        // Show any specific help for selected contentTheme
-        var help = document.getElementById(theme + '-help');
-        if (help) help.style.display = 'block';
```

Comment:
@Maneet-k Why did you remove this code in the latest commit?

Reply:
I see you've moved the code -- kindly explain why it was necessary.

Reply:
I moved the code cuz if I haven't moved it that invalid contentTheme would've applied and the jquery was also breaking because of that so I moved this before the content theme gets applied and I could change the invalid contenttheme value to null(auto's light case). this is all done to get the help displayed on the DOM

Reply:
And I agree I didn't need to move the code because I could set the contentTheme to null after the help in this code too.